generic-styles/content-common.less

```diff
@@ -364,6 +364,16 @@ blockquote {
     }
   }
 
+  [data-type="problem"] {
+    display: flex;
+    align-items: baseline;
+
+    .os-problem-container {
+      display: grid;
```

Comment:
I'm not sure why `display: grid` fixes the issue, but I think it's worth noting that setting it makes its contents to be in rows, which could be undesirable if `.os-problem-container` had some elements that were supposed to fit in one line.

Currently, It's not the case, but I think it's not unreasonable for a `span` to be inside it.